src/pages/Portfolio/ImpactDashboard/DistributionGraphs.vue

```diff
@@ -94,10 +110,10 @@ export default {
 		fetchLocationStats() {
 			if (this.locationLoading && !this.locationLoadingPromise) {
 				this.locationLoadingPromise = this.apollo.query({
-					query: gql`query myStatsByCountry {
+					query: gql`query myStatsByCountry ($onlyActiveLoans: Boolean) {
```

Comment:
We'll need to make sure that the monolith releases to prod before we merge the gql changes to main.